docs/ReleaseNotes.md

```diff
@@ -30,7 +30,7 @@ Additionally, builds for the project now require JDK 11. The project is still ta
 * **Performance** Improvement 3 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
 * **Performance** Improvement 4 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
 * **Performance** Improvement 5 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
-* **Feature** Feature 1 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
+* **Feature** Avoid repeating index endpoints conflicts [(Issue #1015)](https://github.com/FoundationDB/fdb-record-layer/issues/1015)
```

Comment:
This is supposed to be Issue number not PR number. You could create an Issue from https://github.com/FoundationDB/fdb-record-layer/issues/new/choose